examples/lib/memorymonitor.ex

```diff
@@ -0,0 +1,33 @@
+defmodule MemoryMonitor do
+  use Honey, license: "Dual BSD/GPL"
+
+  @max_entries 41943040
+  defmap(:mmap_count, :bpf_array, [max_entries: @max_entries])
+  defmap(:munmap_count, :bpf_array, [max_entries: @max_entries])
+  defmap(:brk_count, :bpf_array, [max_entries: @max_entries])
+
+  @sec "tracepoint/raw_syscalls/sys_enter"
+  def main(ctx) do
+    syscall_id = ctx.id
+    pid = Honey.BpfHelpers.bpf_get_current_pid_tgid()
```

Comment:
Instead of always referencing the full module path, it would be much better to use the `alias Honey.BpfHelpers as: BpfHelper` and then use `BpfHelper.some_function`.